src/lib.rs

```diff
@@ -135,7 +136,9 @@ impl IOSurface {
                                                   mem::transmute(self.as_concrete_TypeRef()),
                                                   0);
 
-            assert_eq!(gl_error, kCGLNoError);
+            if gl_error != kCGLNoErrer {
+                panic!(&*CStr::from_ptr(CGLErrorString(gl_error)).to_string_lossy());
```

Comment:
But no assertion in rust-layers is being triggered. [I'm hitting a platform-specific error](https://github.com/servo/servo/pull/8641#issuecomment-177631392).